### PR TITLE
[5.1] Increase size of buffer for preferences to allow for upto 105K NVTs.

### PR DESCRIPTION
**Pull request by jjnicola**
Backport of https://github.com/greenbone/openvas/pull/432

### CODE REVIEW

File: src/ntp.c [hunk 1]
```diff
@@ -209,6 +211,17 @@ ntp_read_prefs (int soc)
...
 
+  if (buffer_size_count >= input_sz) /* finished = 1; */
+    {
+      log_write ("%s: The received plugin list is longer than expected.\n",
+                 __func__);
+    }
+
+#if DEBUGMORE
+  log_write ("%s: Received %d chars from the client as plugin list.\n",
+￼            __func__, buffer_size_count);
```
Comment (bjoernricks) [inline src/ntp.c]: Could it be possible that the line contains some strange character at the beginning?

Comment (jjnicola) [reply src/ntp.c]: @bjoernricks,
I couldn't see the char, but removing all the spaces in that line and adding it again, there was a difference. So, pushing again with the invisible char.

File: src/ntp.c [hunk 2]
```diff
@@ -179,6 +180,7 @@ ntp_read_prefs (int soc)
           exit (0);
         }
 
+      buffer_size_count =  buffer_size_count + n;
```
Comment (cfi-gb) [inline src/ntp.c]:
```suggestion
      buffer_size_count = buffer_size_count + n;
```

File: CHANGES [hunk 3]
```diff
@@ -8,6 +8,11 @@ $ git clone https://github.com/greenbone/openvas-scanner.git
 $ cd openvas-scanner && git checkout openvas-scanner-5.1 && git log
 
 
+openvas-scanner 5.1.3 (unreleased)
+
+* Increase size of buffer for preferences to allow for upto 105K NVTs.
```
Comment (cfi-gb) [inline CHANGES]:
```suggestion
* Increase size of buffer for preferences to allow for up to 105K NVTs.
```

**Pull request merged**